e2e-tests/portal/cypress/support/portal/createBssApplicationReadyForCheck.js

```diff
@@ -0,0 +1,81 @@
+const dashboard = require('../../e2e/pages/dashboard');
+const selectScheme = require('../../e2e/pages/selectScheme');
+const beforeYouStart = require('../../e2e/pages/beforeYouStart');
+const bankDetails = require('../../e2e/pages/bankDetails');
+const contract = require('../../e2e/pages/contract');
+const aboutSupplier = require('../../e2e/pages/contract/about/about-supplier');
+const aboutBuyer = require('../../e2e/pages/contract/about/about-buyer');
+const aboutFinancial = require('../../e2e/pages/contract/about/about-financial');
+const eligibilityCriteria = require('../../e2e/pages/eligibilityCriteria');
+const eligibilityPreview = require('../../e2e/pages/eligibilityPreview');
+const bondDetails = require('../../e2e/pages/bondDetails');
+const bondFinancialDetails = require('../../e2e/pages/bondFinancialDetails');
+const bondFeeDetails = require('../../e2e/pages/bondFeeDetails');
+const contractReadyForReview = require('../../e2e/pages/contract/contract-readyForReview');
```

Comment:
For all the pages imports, could do this instead with 1x import/require 😎 

```js
const {
  dashboard,
  selectScheme,
  beforeYouStart,
  bankDetails,
  contract,
  aboutSupplier,
  // ...
} = require('../../e2e/pages');
```

Reply:
Imported as requested.